model/project_parser.go

```diff
@@ -1130,7 +1132,11 @@ func evaluateBVTasks(tse *taskSelectorEvaluator, tgse *tagSelectorEvaluator, vse
 				isGroup = true
 			}
 		}
-		if err1 != nil && err2 != nil {
+		if pbvt.TaskGroup != nil {
+			names = append(names, pbvt.Name)
```

Comment:
Can you explain why we add name specifically here? (Would this not be covered by 1125 or 1129)?

Reply:
The above checks on 1125 and 1129 are checking the defined `tasks` and `task_groups` on the config.  Since now we can define a whole new task group inline in the build variants, the name of that group won't be found in either the `tasks` or `task_groups`. So to ensure these inline groups get added to the names list I've added this if statement.